avm/utilities/pipelines/sharedScripts/Set-ModuleReadMe.ps1

```diff
@@ -1158,11 +1158,17 @@ function Set-UsageExamplesSection {
     # Prepare data (using thread-safe multithreading) to consume later
     $buildTestFileMap = [System.Collections.Concurrent.ConcurrentDictionary[string, object]]::new()
     $testFilePaths | ForEach-Object -Parallel {
+        $dict = $using:buildTestFileMap
+
         $folderName = Split-Path (Split-Path -Path $_) -Leaf
-        $buildTemplate = (bicep build $_ --stdout 2>$null) | ConvertFrom-Json -AsHashtable
+        $builtTemplate = (bicep build $_ --stdout 2>$null) | Out-String
 
-        $dict = $using:buildTestFileMap
-        $null = $dict.TryAdd($folderName, $buildTemplate)
+        if ([String]::IsNullOrEmpty($builtTemplate)) {
+            throw "Failed to build template [$_]. Try running the command ``bicep build $_ --stdout`` locally for troubleshooting."
```

Comment:
Suggesting the additional sentence for local troubleshooting as we have latest Bicep enforced on runners 
```suggestion
            throw "Failed to build template [$_]. Try running the command ``bicep build $_ --stdout`` locally for troubleshooting. Make sure you have the latest Bicep CLI installed."
```

Reply:
@eriqua this is not actually relevant to this error :D But I guess it can't hurt ;)

Reply:
Why not? How can you troubleshoot an issue with 2 different versions of the tools?